k8s/airflow-operator/deployer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst
```

Comment:
Please point to an exact version (v0.6.2 in other apps in this repo).

Reply:
done.